src/app/mesh/qgsmaptooleditmeshframe.cpp

```diff
@@ -92,6 +93,8 @@ QgsMapToolEditMeshFrame::QgsMapToolEditMeshFrame( QgsMapCanvas *canvas )
   mActionDigitizing->setCheckable( true );
 
   mActionRemoveVerticesFillingHole = new QAction( this );
+  mActionDelaunayTriangulation = new QAction( tr( "Delaunay triangulation with selected vertices" ), this );
+  mActionRefinement = new QAction( tr( "Refine current face" ), this );
```

Comment:
mActionRefinement -> mActionFacesRefinement

isn't it for all faces selected?

Reply:
ok for renaming.

Depends of how many face selected:
0 --> face under mouse cursor ("current face")
1 --> "Refine selected face"
>1 --> "Refines X selected faces"
Here it is the default text (at the beginning, no face selected).
The text is changed later depending of how many face are selected